pkg/cmd/util/net.go

```diff
@@ -123,8 +124,9 @@ func TransportFor(ca string, certFile string, keyFile string) (http.RoundTripper
 	}
 
 	// Copy default transport
-	transport := *http.DefaultTransport.(*http.Transport)
-	transport.TLSClientConfig = &tls.Config{}
+	transport := knet.SetTransportDefaults(&http.Transport{
```

Comment:
the intent of this was to pick up default transport settings, which the lib does safely